pkg/controller/csi_handler_test.go

```diff
@@ -832,28 +823,28 @@ func TestCSIHandler(t *testing.T) {
 			},
 		},
 		{
-			name:           "Node without annotations -> error",
-			initialObjects: []runtime.Object{pvWithFinalizer(), nodeWithoutAnnotations()},
+			name:           "Node without CSINode -> error",
```

Comment:
Can you please add test case where Node has the annotation and CSINode does not exists? It should fail, same as this tests. Just to make sure we really do not process the annotation accidentally.

Reply:
Sure, will check.